internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -305,27 +300,20 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 				},
 			},
 			"termination_protection_enabled": schema.BoolAttribute{
-				Computed: true,
-				Optional: true,
-				PlanModifiers: []planmodifier.Bool{
-					boolplanmodifier.UseStateForUnknown(),
-				},
+				Computed:            true,
+				Optional:            true,
 				MarkdownDescription: "Flag that indicates whether termination protection is enabled on the cluster. If set to `true`, MongoDB Cloud won't delete the cluster. If set to `false`, MongoDB Cloud will delete the cluster.",
 			},
 			"version_release_system": schema.StringAttribute{
-				// TODO: probably leave validation just in the server, ValidateFunc: validation.StringInSlice([]string{"LTS", "CONTINUOUS"}, false),
-				Computed: true,
-				Optional: true,
-				PlanModifiers: []planmodifier.String{
```

Comment:
All UseStateForUnknown are removed.
Although they simplify the plan output, they can also lead to:
`Error: Provider produced inconsistent result after apply`

I think a better option is using the resource_plan modifier to support simplifying the plan when we are absolutely sure there will be no inconsistent result after apply